clients/ops/admin-ui/src/app/store.ts

```diff
@@ -38,14 +73,33 @@ const reducer = {
   connectionType: connectionTypeReducer,
 };
 
+const rootReducer = (state: any, action: AnyAction) => {
+  if (action.type === "auth/logout") {
+    storage.removeItem(STORAGE_ROOT_KEY);
+    // eslint-disable-next-line no-param-reassign
+    state = {};
+  }
+  return combineReducers(reducer)(state, action);
```

Comment:
I think this will re-combine the reducer object on every action. Wrapping the reducer once could be easier to follow: `const reducer = combineReducers({ .. })`